cmd/oci-image-tool/create.go

```diff
@@ -40,9 +41,10 @@ func createHandle(context *cli.Context) error {
 	}
 
 	v := bundleCmd{
-		typ:  context.String("type"),
-		ref:  context.String("ref"),
-		root: context.String("rootfs"),
+		typ:      context.String("type"),
+		ref:      context.String("ref"),
+		root:     context.String("rootfs"),
+		platform: context.StringSlice("platform"),
```

Comment:
So, don't explain it to me: document the flag in the code and help functions. Based on this, I have no clue how to use it. Platform is a nested structure and that needs to represented as a filter here.